python/dapper_python/dataset_loader.py

```diff
@@ -0,0 +1,445 @@
+"""
+dataset_loader.py - A module for discovering and loading SQLite databases from XDG directories
+
+This module provides both a library interface and a command line interface.
+"""
+
+import os
+import sys
+import sqlite3
+import logging
+import argparse
+import shutil
+import xdg.BaseDirectory

```

Comment:
I don't think we can use this module, since the paths it returns won't match the ones that are given by the (not quite equivalent) Rust library -- particularly on Windows and macOS, the paths will not match.